test/test-init.js

```diff
@@ -91,6 +110,9 @@ module.exports = {
     gnosisProxy,
     seedFactory,
     seedMasterCopy,
+    deployLBPFactory,
```

Comment:
also the naming decision here is not the best, should be something like `LBPFactory` in this case

Reply:
Done